modules/src/mock/context.rs

```diff
@@ -124,19 +128,60 @@ pub struct MockContext {
 
     /// ICS26 router impl
     router: MockRouter,
+
+    /// Object capabilites impl
+    ocap: Arc<Mutex<MockOCap>>,
 }
 
 /// Returns a MockContext with bare minimum initialization: no clients, no connections and no channels are
 /// present, and the chain has Height(5). This should be used sparingly, mostly for testing the
 /// creation of new domain objects.
 impl Default for MockContext {
     fn default() -> Self {
+        let ocap = Arc::new(Mutex::new(MockOCap::default()));
         Self::new(
             ChainId::new("mockgaia".to_string(), 0),
             HostType::Mock,
             5,
             Height::new(0, 5),
         )
+        .with_router(dummy_router(ocap.clone()))
```

Comment:
The default implementation now comes with a `dummy_router()` and an ocap system where the default port is already owned by the `DummyModule`. This setup simplifies handler tests.